Bonsai.NuGet.Design/PackageSourceConfigurationDialog.Designer.cs

```diff
@@ -29,7 +29,6 @@ protected override void Dispose(bool disposing)
         private void InitializeComponent()
         {
             this.components = new System.ComponentModel.Container();
-            System.ComponentModel.ComponentResourceManager resources = new System.ComponentModel.ComponentResourceManager(typeof(PackageSourceConfigurationDialog));
```

Comment:
Oops, yes I did, I was copying the pattern from another place, so I will check both, and try to figure out another pattern since I know the .NET core forms designer tends to be more fragile.